VideoLocker/res/layout/view_register_spinner.xml

```diff
@@ -3,32 +3,22 @@
     xmlns:tools="http://schemas.android.com/tools"
```

Comment:
@BenjiLee This is not abstract enough to be general spinner. There is a custom error layout here.